assets/js/util/sanitize.test.js

```diff
@@ -29,3 +29,19 @@ describe( 'sanitizeHTML', () => {
 		} );
 	} );
 } );
+
+describe( 'unTrailingSlashIt', () => {
+	it( 'should not change a URL without trailing slash', () => {
+		const url = 'http://example.org';
+		expect( unTrailingSlashIt( url ) ).toEqual( 'http://example.org' );
+	} );
```

Comment:
Defining the constant here doesn't add much to the test, I would say it actually impedes readability here. Let's make these all one-liners to make them easier to scan 🙂 
```suggestion
	it( 'should not change a URL without trailing slash', () => {
		expect( unTrailingSlashIt( 'http://example.org' ) ).toEqual( 'http://example.org' );
	} );
```